exp/MOM_compile.csh

```diff
@@ -260,8 +264,12 @@ if( $type == ACCESS-OM || $type == ACCESS-ESM) then
     set srcList = ( $srcList access/shared )
 endif
 
```

Comment:
I think this should be bracketed in logic that checks for an external FMS too.

Reply:
src/access/shared/gtracer_flux.F90:
```
module gtracer_flux_mod
```

It looks like it's required:
```
$ rg 'use gtracer_flux_mod' *
src/access/accesscm_coupler/mom_oasis3_interface.F90
97:use gtracer_flux_mod, only: set_coupler_type_data, extract_coupler_type_data

src/access/accesscm_coupler/ocean_solo.F90
158:  use gtracer_flux_mod,         only: flux_exchange_init, atmos_ocean_fluxes_calc
159:  use gtracer_flux_mod,         only: gas_fields_restore, gas_fields_restart

src/access/accessom_coupler/ocean_solo.F90
115:  use gtracer_flux_mod,         only: flux_exchange_init, atmos_ocean_fluxes_calc
116:  use gtracer_flux_mod,         only: gas_fields_restore, gas_fields_restart

src/access/accessom_coupler/mom_oasis3_interface.F90
109:use gtracer_flux_mod, only: set_coupler_type_data
```